src/layout/SigningStatusPanel/PanelNoActionRequired.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
```

Comment:
I know me might disagree on this, but it would be nice for the entire code base to be consistently using `AppRoutingContext` for these things instead of react router directly.

Reply:
You're right, we do disagree on this 🙈  I don't see that it would gain such performance benefits that it outweighs the issues of added complexity, maintenance required and the fact that this can hide bugs that would be hard to resolve. How many times does the URL actually change? The most changes that I can see is from task page to task page. And shouldn't the components then rerender anyways because the layout is different?

Reply:
Yep, maybe it doesn't change all that much. When I implemented this, there was a very noticeable difference in performance, but that was probably also mostly because of hot paths (which this isn't). The main problem I have with using `react-router-dom` directly now is not necessarily because of performance or re-renders, but because of consistency in the code base. Having two ways of doing one thing in the same code base is a 'smell'.